data/zone.schema.ts

```diff
@@ -0,0 +1,42 @@
+import mongoose from 'mongoose';
+import { Schema } from 'mongoose';
+
+enum Status {
+    todo = 'To Do',
```

Comment:
enum keys are usually written in all uppercase, due to being finally defined and accessed in a static manner